src/libs/auth.js

```diff
@@ -162,7 +162,7 @@ authStore.subscribe(async (state, oldState) => {
   if (!oldState.isSignedIn && state.isSignedIn) {
     window.newrelic.setCustomAttribute('userGoogleId', state.user.id)
     window.Appcues && window.Appcues.identify(state.user.id, {
-      dateJoined: await getDateJoined()
+      dateJoined: state.registrationStatus === 'registered' ? await getDateJoined() : Date.now()
```

Comment:
TOL: Would it make more sense to omit sending appcues any data for a test user at all? It feels risky to me to be adding a logic specifically for a test user in the main code as it could have unintended consequences/make it less clear what should be happening. (not sure if that's feasible either but just wanted to ask) Or could we more specifically check if it's a test user before changing what is sent? Essentially just checking if the user is registered seems potentially risky/loose?

Reply:
This logic isn't specifically for test users, it's for any unregistered user.